kloppy/infra/serializers/tracking/metrica.py

```diff
@@ -175,6 +179,11 @@ def deserialize(
         # consider reading this from data
         frame_rate = 25
 
+        # TODO: also used in Tracab, extract to a method
+        home_team = Team(team_id="home", name="home", ground=Ground.HOME)
+        away_team = Team(team_id="away", name="away", ground=Ground.AWAY)
+        teams = [home_team, away_team]
```

Comment:
@koenvo I looked a bit into creating the Players and adding them to the Team for Metrica files, and probably the same applies to Tracab. But I have a doubt regarding how to do it. 

To create the players I need to access the first two rows of the file, but as the file is consumed in the iterator, I don't know how to get them out to create the players for each team without adding them to all the frames. 

I could read the two first lines outside of `__create_iterator` but that's probably not the right way to do it? I think I should have time this week to add all the players to all other data formats other than EPTS, but might need a bit of help finding out the best way to do it.

Reply:
Implemented.